Mathlib/Data/LocallyFinsupp.lean

```diff
@@ -101,6 +101,20 @@ lemma apply_eq_zero_of_not_mem [Zero Y] {z : X} (D : locallyFinsuppWithin U Y)
     (hz : z ∉ U) :
     D z = 0 := nmem_support.mp fun a ↦ hz (D.supportWithinDomain a)
 
+/--
+On a T1 space, the support of a functions with locally finite support within `U` is discrete within
+`U`.
+-/
+theorem discreteWithinSupport [Zero Y] [T1Space X] (D : locallyFinsuppWithin U Y) :
```

Comment:
```suggestion
theorem eq_zero_codiscreteWithin [Zero Y] [T1Space X] (D : locallyFinsuppWithin U Y) :
```
Don't try to be too creative with names, it only makes the lemmas harder to discover :-)